package/src/components/AttachmentPicker/AttachmentPicker.tsx

```diff
@@ -102,9 +134,28 @@ const renderImage = ({
     maxNumberOfFiles,
     numberOfAttachmentPickerImageColumns,
     selected,
+    setSelectedFiles,
     setSelectedImages,
   } = item;
-  const onPress = () => {
+
+  const contentType = lookup(asset.filename) || 'multipart/form-data';
+
+  const isImage = asset.filename
+    ? contentType.startsWith('image/')
+      ? 'image'
+      : 'video'
+    : asset.type === 'video'
+    ? 'video'
+    : 'image';
+  const videoDuration = asset.duration ? asset.duration : asset.playableDuration;
+
+  const duration = videoDuration
+    ? videoDuration / 3600 >= 1
+      ? dayjs.duration(videoDuration, 'second').format('HH:mm:ss')
+      : dayjs.duration(videoDuration, 'second').format('mm:ss')
+    : null;
```

Comment:
can we rewrite the first and second of these to avoid nested ternaries? Even if it means extracting it into two small functions or something, would be easier readable